frontend/game-rating-platform/src/compoments/GameTable.tsx

```diff
@@ -23,21 +16,42 @@ export function GameTable({
   // cache original game data for filter operations
   const [origGameData, setOrigGameData] = useState<GameProps[]>([]);
   const [gameData, setGameData] = useState<GameProps[]>([]);
+  const [activeModal, setActiveModal] = useState<boolean>(false);
+  const [clickedGame, setClickedGame] = useState<GameProps>({
+    id: 0,
+    title: "",
+    releaseDate: "",
+    developers: [],
+    description: "",
+    trailer: "",
+    avgRating: 0,
+    platform: "",
+  });
+
+  function openModal(e: SyntheticEvent<HTMLTableRowElement, Event>) {
+    setActiveModal(true);
+    let filteredData = gameData.filter(
+      (prop) => prop.title === e.currentTarget.id
```

Comment:
Ich bin zwar nicht als Reviewer eingetragen, aber bin zufällig über das Repository gestolpert, deswegen gebe ich hier auch mal meinen Senf dazu: Um ein einzelnes Element in einem Array zu finden, gibt es die Methode `.find()` - dass du hier über die ID suchst, lässt mich vermuten, dass du genau das tun willst. `.filter()` und dann das erste Element nehmen ist mMn in diesem Fall ein Umweg.